src/api/services/configs.ts

```diff
@@ -134,3 +134,34 @@ export const getRelatedConfigs = async (configID: string) => {
   );
   return res.data;
 };
+
+export type ConfigTypeItem = {
+  config_type: string;
+};
+
+export const getConfigsTypes = async () => {
+  const res = await IncidentCommander.get<ConfigTypeItem[]>(`/config_types`);
```

Comment:
in case backend api errors data will be we need to handle those scenarios here.

Reply:
@Prashanth-Nelli  throw an error in this case? Something like i`f(!data) throw error?`

Reply:
As we are using this in react queries, I think we need to handle cases where we are not getting the data. If it is already handled then we are good.

Reply:
Just updating the types to reflect the reality that the data can be null.